unicodetools/data/ucd/dev/NamesList.txt

```diff
@@ -13711,6 +13711,8 @@
 	* Kyrgyzstan
 20C1	SAUDI RIYAL SIGN
 	* Saudi Arabia
+20C3	UAE DIRHAM SIGN
+	* United Arab Emirates
```

Comment:
Ken _generates_ the NamesList from other sources, and likes to add change comments at the top. I assume that he would prefer not to add this in this kind of PR.

Reply:
Yes, NamesList should not be touched by these PRs.

Reply:
I agree with Robin. Don't touch NamesList.txt directly. (And don't deal with Index.txt, either -- updating that is outside the context of establishing the properties.)